openwisp_firmware_upgrader/admin.py

```diff
@@ -325,9 +325,15 @@ def get_queryset(self, request, select_related=True):
         return qs
 
 
+# TODO:
+# DeviceAdmin.inlines += [DeviceFirmwareInline]
+# DeviceAdmin.conditional_inlines += [DeviceUpgradeOperationInline]
```

Comment:
I think you can remove this.

Reply:
I was there to remember that solution is better, but I have doubs